CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.48.0
+* Enhancement - Docker stop timeout buffer increased from 30s to 2m [2697](https://github.com/aws/amazon-ecs-agent/pull/2697)
+* Enhancement - More informative ENI attachment logs [2703](https://github.com/aws/amazon-ecs-agent/pull/2703)
+* Enhancement - The environment variable ECS_PULL_DEPENDENT_CONTAINERS_UPFRONT=false as default [2731](https://github.com/aws/amazon-ecs-agent/pull/2731) 
+* Bug - Fix a bug where agent persists RCI client token to avoid being registered as different container instance ARNs [2708](https://github.com/aws/amazon-ecs-agent/pull/2708)
+* Bug - Fix jumbled min & max for engine connection retry delays [2721](https://github.com/aws/amazon-ecs-agent/pull/2721)
+
```

Comment:
there needs to be another changelog: 
* Enhancement - Add pulled containers metadata to Task Metadata Endpoint V4.